tests/ert/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -465,3 +474,26 @@ async def test_ensure_multi_level_events_in_order(evaluator_to_use):
                 if "reals" in event.snapshot:
                     assert ensemble_state == ENSEMBLE_STATE_STARTED
                 ensemble_state = event.snapshot.get("status", ensemble_state)
+
+
+@given(
+    num_cpu=st.integers(min_value=1),
+    start=st.datetimes(),
+    end=st.datetimes(),
+    cpu_seconds=st.floats(min_value=0),
+)
+def test_overspent_cpu_is_logged(
+    num_cpu: int,
+    start: datetime.datetime,
+    end: datetime.datetime,
+    cpu_seconds: float,
+):
+    message = detect_overspent_cpu(
+        num_cpu, "dummy", FMStepSnapshot(start=start, end=end, cpu_seconds=cpu_seconds)
+    )
+    duration = (end - start).total_seconds()
+    print(f"{duration=} {cpu_seconds/duration=} {num_cpu=}")
+    if duration > 0 and cpu_seconds / duration > num_cpu:
```

Comment:
So if duration is `1`, and cpu_seconds is `1`, and incidentally num_cpu is `1` .. ?

Reply:
then no warning should be printed.

Reply:
Indeed. 🤦